Essentials/src/main/java/com/earth2me/essentials/commands/Commandrecipe.java

```diff
@@ -53,7 +53,23 @@ public void run(final Server server, final CommandSource sender, final String co
             throw new NotEnoughArgumentsException();
         }
 
-        final ItemStack itemType = ess.getItemDb().get(args[0]);
+        final ItemStack itemType;
+
+        if (args[0].equalsIgnoreCase("hand")) {
+            final User user = ess.getUser(sender.getPlayer());
+            if (user == null) {
+                throw new Exception(tl("recipeConsoleNoHands"));
+            }
+            final Material item = user.getItemInHand().getType();
+
+            if (item.isAir()) {
+                throw new Exception(tl("recipeCannotBeAir"));
```

Comment:
`recipeNone` already exists, this shouldn't need a more specific message

Reply:
Actually, if you leave it as air I'm pretty sure it would just end up hitting the condition below anyway, so this check might not even be necessary